policy/policy.go

```diff
@@ -556,3 +559,27 @@ func IsPolicyMrn(candidate string) error {
 	}
 	return nil
 }
+
+func (s *ScoringSystem) UnmarshalJSON(data []byte) error {
+	// check if we have a number
+	var code int32
+	err := json.Unmarshal(data, &code)
+	if err == nil {
+		*s = ScoringSystem(code)
+	} else {
+		var name string
+		_ = json.Unmarshal(data, &name)
+
+		switch name {
+		case "lowest":
```

Comment:
Why not "worst"?

Reply:
Sounded like "worst scoring system". I've always regretted that name ever since choosing it, but I may also be wrong about it...

Reply:
Changed based on more feedback, thank you for raising it!